train.py

```diff
@@ -122,14 +122,15 @@ def train(hyp, opt, device, callbacks):  # hyp is path/to/hyp.yaml or hyp dictio
         with torch_distributed_zero_first(LOCAL_RANK):
             weights = attempt_download(weights)  # download if not found locally
         ckpt = torch.load(weights, map_location='cpu')  # load checkpoint to CPU to avoid CUDA memory leak
-        model = Model(cfg or ckpt['model'].yaml, ch=3, nc=nc, anchors=hyp.get('anchors')).to(device)  # create
+        model = Model(cfg or ckpt['model'].yaml, ch=hyp.get('ch'), nc=nc,
```

Comment:
I suppose this should default to 3 for backward compatibility:
```suggestion
        model = Model(cfg or ckpt['model'].yaml, ch=hyp.get('ch', 3), nc=nc,
```

Reply:
i suppose this code defaults ch to 3 when hyp.get has nothing? That seems good to me :)